src/assets/translations/en/main.json

```diff
@@ -156,26 +156,21 @@
         }
       },
       "getStarted": {
-        "header": "Stake your %{assetName} to earn %{apr}% APR*",
+        "header": "Stake your %{assetName} to earn",
         "body": "Staking is the process of holding funds in a cryptocurrency wallet to support the operations of a blockchain network and, in return, holders are rewarded for their contribution.",
         "userProtectionInfo": "*Returns are estimated and fluctuate with market conditions.",
         "cta": {
           "startStaking": "Start Staking",
           "learnMore": "Learn More"
         }
       },
-      "staking": {
-        "estimateDisclaimer": "*Estimates fluctuate with market conditions.",
-        "byContinuing": "By continuing, you are accepting the",
-        "risks": "risks",
-        "ofParticipating": "of participating in this feature and agreeing to the",
-        "terms": "terms",
-        "stakingYourTokens": "Staking your tokens",
-        "unstakingYourTokens": "Unstaking your tokens",
-        "tooltip": {
-          "validator": "The Validator is responsible for processing transactions, storing data and adding blocks to the blockchain.",
-          "gasFees": "Gas fees on %{networkName} are much lower than those on Ethereum Mainnet. This fee is paid directly to the validators that secure and operate the network and is not paid to ShapeShift."
-        }
+      "cosmosOverview": {
+        "rewards": "The rewards you earn will accrue automatically.",
+        "cta": "Start Staking",
+        "learnMore": "Learn More",
+        "withdrawals": "Unstaking",
+        "emptyWithdraws": "Pending unstakings will show here.",
+        "availableDate": "available %{date}"
       },
       "foxyOverview": {
         "header": "Deposit FOX to get FOXy to earn",
```

Comment:
Request change to 'Deposit FOX to receive and earn FOXy'

Reply:
That's not within the diff of this PR, but that's already how the final string looks like ser ✅

Reply:
That's outside of the scope of this PR, but I'll get a separate one spinned up for ya!